gnomad_qc/v4/assessment/calculate_per_sample_stats.py

```diff
@@ -449,28 +510,18 @@ def main(args):
         raise ValueError("Stratifying by subset is only working on exomes data type.")
 
     try:
-        if args.create_per_sample_counts_ht:
-            chrom = "chr22" if test else None
-            if data_type == "exomes":
-                logger.info("Calculating per-sample variant statistics for exomes...")
-                mt = get_gnomad_v4_vds(
-                    test=test, release_only=True, split=True, chrom=chrom
-                ).variant_data
-            else:
-                logger.info("Calculating per-sample variant statistics for genomes...")
-                mt = get_gnomad_v4_genomes_vds(
-                    test=test, release_only=True, split=True, chrom=chrom
-                ).variant_data
-                ukb_capture_intervals = False
-                broad_capture_intervals = False
-
+        if args.create_filter_group_ht:
+            logger.info("Creating Table of filter groups for summary stats...")
             release_ht = release_sites(data_type=data_type).ht()
-            if test:
-                release_ht = hl.filter_intervals(
-                    release_ht, [hl.parse_locus_interval("chr22")]
-                )
+            filtering_groups_res = get_summary_stats_filtering_groups(
+                data_type=data_type, test=test, autosomes_only=autosomes_only
+            )
+            if args.test_n_partitions:
+                release_ht = release_ht._filter_partitions(test_partitions)
+            if autosomes_only:
+                release_ht = filter_to_autosomes(release_ht)
```

Comment:
I think it's still good to filter release_ht for test dataset, so the the test would go faster: 
```suggestion
            if args.test_dataset:
                release_ht = hl.filter_intervals(
                    release_ht, [hl.parse_locus_interval("chr22")]
                )
```

Reply:
I want to be able to test the full test dataset, for the faster test, use `--test-dataset` and `--test-n-partitions n`